dune

```diff
@@ -1,11 +0,0 @@
-(subdir
```

Comment:
Let's add `dune` file with `node_modules` exclusion + explanatory comment, similar to what we discussed on the opam template:

https://github.com/melange-re/melange-opam-template/pull/13#discussion_r1212075219